src/rest_api.c

```diff
@@ -110,19 +116,30 @@ int auth_handler(struct mg_connection * conn, void * cbdata)
 struct mg_context * start_server()
 {
     struct mg_context *ctx;
+    ecc_init();
+
+    const char *pem_file_path = "../../bits/public_key.pem";
+    char *pem_public_key = read_file_content(pem_file_path);
```

Comment:
memory leak?

Reply:
You mean for pem_public_key right? Because we have free(public_key) in stop_server. 
I will add free(pem_public_key) right after usage of it.

Reply:
yes, it's about `pem_public_key`